download-l1b.sh

```diff
@@ -61,23 +63,15 @@ then
   echo "ERROR:download-l1b.sh: file $SECRETFILE missing: create this file with your PO.DAAC username and password, each in one single line."
   exit 3
 fi
-USERNAME=$(head -n1 $SECRETFILE)
-PASSWORD=$(tail -n1 $SECRETFILE)
+USERNAME=$(head -n1 "$SECRETFILE")
+PASSWORD=$(tail -n1 "$SECRETFILE")
 
 echo "$MSG"
 
 #fetch the data
-mkdir -p $LOCALDIR || exit $?
+mkdir -p "$LOCALDIR" || exit $?
 wget \
   --user=$USERNAME \
   --password=$PASSWORD \
-  --recursive \
-  --timestamping \
-  --continue \
-  --no-parent \
-  --no-directories \
-  --accept "$TAR_FILE" \
-  --show-progress \
-  --verbose \
```

Comment:
I'm not sure why you deleted these wget options, can you clarify?

Reply:
It does not work on my side. I was asked to check `wget --help`

Reply:
weird... what version of wget you have?
```
$wget --version
GNU Wget 1.20.3 built on darwin19.0.0.

-cares +digest -gpgme +https +ipv6 +iri +large-file -metalink +nls
+ntlm +opie -psl +ssl/openssl
...
```

Reply:
```
 $wget --version
GNU Wget 1.14 built on linux-gnu.
```
I do not have the tool` tree` but the directories are the same `/L1B/JPL/RL03/2010`

Reply:
OK, leave it without like you edited. I'll check on my side. Those additional arguments are probably unnecessary.